### PR TITLE
Few more fixes

### LINKED ISSUE #4974: Smiley More Box Formatting
When configured as 'popup' smileys, they show up in a special window when you press the MORE button.  

This special window has issues, & needs some help on the .css front...  

The MORE box is too big and doesn't fit:
moreboxtoobig
moreboxtoobig2

Depending on your zoom setting, it disappears altogether:
moreboxhidden

Also, would it be possible to get the MORE button itself on the same line as the smileys, to the right?  It uses a lot of real estate:
morecenter

### PR DESCRIPTION
**Pull request by SychO9**
#4974 #4965

### CODE REVIEW

File: Themes/default/scripts/script.js [hunk 1]
```diff
@@ -1417,8 +1417,7 @@ function smfSelectText(oCurElement, bActOnElement)
 		// Safari is special!
 		if (oCurSelection.setBaseAndExtent)
 		{
-			var oLastChild = oCodeArea.lastChild;
-			oCurSelection.setBaseAndExtent(oCodeArea, 0, oLastChild, 'innerText' in oLastChild ? oLastChild.innerText.length : oLastChild.textContent.length);
+			oCurSelection.setBaseAndExtent(oCodeArea, 0, oCodeArea, oCodeArea.childNodes.length);
```
Comment (jdarwood007) [inline Themes/default/scripts/script.js:1420]: What are we fixing here?

**Comment by jdarwood007** [comment]
FYI, when you send up your commits, use Fixes #### (see: https://blog.github.com/2013-05-14-closing-issues-via-pull-requests/)

File: Themes/default/scripts/script.js (hunk 1 above)
Comment (SychO9) [reply Themes/default/scripts/script.js:1420]: #4965

**Comment by SychO9** [comment]
I see, got it

**Review by jdarwood007 — APPROVED** [review]

**Pull request merged**